applications/nrf5340_audio/src/bluetooth/le_audio_cis_headset.c

```diff
@@ -677,14 +677,17 @@ int le_audio_user_defined_button_press(enum le_audio_user_defined_action action)
 	return 0;
 }
 
-int le_audio_config_get(uint32_t *bitrate, uint32_t *sampling_rate)
+int le_audio_config_get(uint32_t *bitrate, uint32_t *sampling_rate, uint32_t *octets_per_frame)
```

Comment:
I think this PR great!
But I'm wondering if we can prevent adding new parameter in 'le_audio_config_get', but just calculate the octets from bitrate when we need. This is something we can discuss.